_data/navigation.yml

```diff
@@ -25,54 +25,71 @@
       link: /calendar
       new_window: false
       external: false
+      img_src: /images/nav-icons/Calendar.svg
     - name: Programs
       link: /programs
       new_window: false
       external: false
+      img_src: /images/nav-icons/Programs.svg
     - name: Mailing Lists
       link: /subscribe
       new_window: false
       external: false
+      img_src: /images/nav-icons/Mail.svg
     - name: Discuss
       link: https://discuss.layer5.io
       new_window: true
       external: true
+      img_src: /images/nav-icons/Discuss.svg
     - name: GitHub
       link: https://github.com/meshery
       new_window: true
       external: true
+      img_src: /images/nav-icons/Github.svg
     - name: Slack
       link: http://slack.layer5.io
       new_window: true
       external: true
+      img_src: /images/nav-icons/Slack.svg
     - name: Youtube
       link: https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1
       new_window: true
       external: true
+      img_src: /images/nav-icons/Youtube.svg
 - name: Resources
   submenu: true
   highlight: false
   subitems:
+    - name: Cloud Native Catalog
```

Comment:
We will probably have a merge conflict here…

Reply:
Oh ok, what can be done @leecalcote